.travis.yml

```diff
@@ -26,3 +26,4 @@ before_install:
   - if [ ! -e "$HOME/.cargo/bin" ]; then curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain $RUST_VERSION -y; fi
   - export PATH="$HOME/.cargo/bin:$PATH"
   - rustup default $RUST_VERSION
```

Comment:
Isn’t this redundant since we set a default in the install?

Reply:
I'm not sure 100% how, but on appveyor, one of the builds where there is a .rustup cache, `rustup update` end up installing the msvc target, which is why I had to add the target to the default. I think we may not be caching wherever this default is being written to.